src/main/scala/edu/ie3/util/geo/GeoUtils.scala

```diff
@@ -0,0 +1,341 @@
+/*
+ * © 2021. TU Dortmund University,
+ * Institute of Energy Systems, Energy Efficiency and Energy Economics,
+ * Research group Distribution grid planning and operation
+*/
+package edu.ie3.util.geo
+
+import edu.ie3.util.exceptions.GeoException
+import org.apache.commons.lang3.ArrayUtils
+import org.locationtech.jts.algorithm.ConvexHull
+import org.locationtech.jts.geom.impl.CoordinateArraySequence
+import org.locationtech.jts.geom.{
+  Coordinate,
+  GeometryCollection,
+  GeometryFactory,
+  LineString,
+  LinearRing,
+  Point,
+  Polygon,
+  PrecisionModel
+}
+import tech.units.indriya.ComparableQuantity
+import tech.units.indriya.quantity.Quantities
+import tech.units.indriya.unit.Units.{METRE, RADIAN}
+
+import java.lang.Math._
+import javax.measure.Quantity
+import javax.measure.quantity.Length
+import scala.collection.immutable.{SortedSet, TreeSet}
+import scala.math.pow
+import scala.util.{Failure, Success, Try}
+
+object GeoUtils {
```

Comment:
Remember to re-introduce function signatures that are used in the old java classes

Reply:
As discussed since we get rid of osmonauts data structures and we would have to reimplement the java code in scala we chose in that case to just get rid of it